lib/doorkeeper/helpers/resource_indicators.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  module Helpers
+    # The RFC for resouce indicators allows for multiple to be specified using this syntax:
+    #  GET /as/authorization.oauth2?response_type=code
+    #  ...
+    #  &scope=calendar%20contacts
+    #  &resource=https%3A%2F%2Fcal.example.com%2F
+    #  &resource=https%3A%2F%2Fcontacts.example.com%2F
+    #
+    # The default ActionController query paramater parsing will not accept that as an array
+    # so we've got to do it ourselves. Post body request parsing also sees the same issue.
+    module ResourceIndicators
+      def self.resource_identifier_from_request(existing_params, request)
```

Comment:
Would it read better if `request` was the first argument given that the method name is `resource_identifier_from_request`?